frontend/src/components/organisms/EventForm.tsx

```diff
@@ -97,24 +267,32 @@ const EventForm = ({ eventType }: EventFormProps) => {
             row
             aria-labelledby="demo-row-radio-buttons-group-label"
             name="row-radio-buttons-group"
-            defaultValue="Virtual"
+            defaultValue={eventDetails ? eventDetails.mode : "VIRTUAL"}
             sx={{ borderRadius: 2, borderColor: "primary.main" }}
           >
             <FormControlLabel
-              value="Virtual"
+              value="VIRTUAL"
               control={<Radio />}
               label={<Typography sx={{ fontSize: 15 }}>Virtual</Typography>}
               onClick={() => radioHandler(0)}
             />
             <FormControlLabel
-              value="In-Person"
+              value="IN_PERSON"
               control={<Radio />}
               label={<Typography sx={{ fontSize: 15 }}>In-Person</Typography>}
               onClick={() => radioHandler(1)}
             />
           </RadioGroup>
         </FormControl>
-        {status == 1 && <LocationPicker label="" />}
+        {status == 1 && (
+          <LocationPicker
+            label=""
+            required={status == 1}
+            name="location"
+            register={register}
+            requiredMessage={errors.location ? "Required" : undefined}
+          />
+        )}
       </div>
       <div className="grid grid-cols-1 sm:grid-cols-2 col-span-2  sm:col-span-1">
         <TextField
```

Comment:
It just occured to me that this should have a `type=number`. Since the input is of type number